pkg/cmd/build/constants.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2024 The Okteto Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package build
+
+const (
+	// DeployOutputModeOnBuild Defines the output mode set to BuildOptions when running a remote deploy
+	DeployOutputModeOnBuild = "deploy"
+	// DestroyOutputModeOnBuild Defines the output mode set to BuildOptions when running a remote destroy
+	DestroyOutputModeOnBuild = "destroy"
```

Comment:
nit: we could also move `OktetoDefaultImageTag` from the `model` pkg, to keep reduce the model's size and responsability / avoid the import/dependency but also because that const is very much in the context of the build/registry

Reply:
It seems easy, but it generates cycle import, we should for sure look for a better place for that constant, but I won't do it as part of this change just because I'm not sure which would be the right place and this PR already has several changes